modules/oxide/src/less/theme/components/menubar/menubar-button.less

```diff
@@ -22,20 +22,20 @@
 @menubar-select-disabled-box-shadow: @toolbar-button-box-shadow;
 @menubar-select-disabled-text-color: fade(@toolbar-button-text-color, 50%);
 
-@menubar-select-hover-background-color: @toolbar-button-hover-background-color;
+@menubar-select-hover-background-color: #285ec7;
```

Comment:
Why where these changed to a hard coded value. If they need altering could they have been altered with fade or something instead.